src/react/6-stateful-hooks.md

```diff
@@ -1,28 +1,28 @@
 @page learn-react/stateful-hooks Stateful Hooks
 @parent learn-react 6
 
-@description Learn how to use hooks in React to manage the state of a component.
+@description Learn how to use React's hooks to manage the state of a component.
 
 @body
 
 ## What are stateful hooks?
 
-Stateful hooks are React's way of managing all of the data that a functional component needs to keep track of. The most widely used stateful hook, `useState`, streamlines the process of both using and updating individual state values in a straightforward way.
+Stateful hooks are React's way of managing the data that a functional component needs to remember. The most widely used stateful hook, [`useState`](https://reactjs.org/docs/hooks-state.html), streamlines the process of both using and updating individual state values in a straightforward way.
 
 Another stateful hook, `useRef`, gives component code access to the underlying DOM elements rendered from the JSX.
 
-Let's take a look at each hook in turn and see what they can do.
+Let's take a look at each hook and see what they can do.
 
 ## useState
 
-When `useState` is called, it returns an array with two elements. The first element is the state value itself; this is the value you would render inside the JSX. The second element is a setter function, which when called, can be used to update the state and trigger a re-render.
+`useState` is called with a single argument, it returns an array with two elements. The argument to `useState` is the default value of the `state` that you are creating. The first element of the return value is the state value itself; this is the value you would render inside the JSX. The second element is a setter function, which when called, can be used to update the state value and trigger a re-render.
```

Comment:
I would move the argument bit to end of the description. I'd argue that describing what it is and does should precede instructions for how to use it.

If you don't like starting the description with "When", I'd suggest rephrase it to the following:

> `useState` accepts a single parameter and returns...

Reply:
I rephrased it to use the `useState` accepts a single parameter, but I'm not sure moving it to the end of the paragraph makes more sense. Maybe you could Slack me to see what you are thinking but I think it would read weird, if I literally moved that first sentence to the end because I make reference to the argument and return values in the following sentences. Gimme a ring, and we'll chat about it.